main.go

```diff
@@ -142,6 +145,7 @@ func main() {
 
 	daemonCommand.PersistentFlags().StringVar(&Flags.Aws.AccessKey.ID, "aws.accesskey.id", "", "ID of the AWS access key")
 	daemonCommand.PersistentFlags().StringVar(&Flags.Aws.AccessKey.Secret, "aws.accesskey.secret", "", "Secret of the AWS access key")
+	daemonCommand.PersistentFlags().StringVar(&Flags.Aws.CertsDir, "aws.certsdir", "", "Certificated to be placed in /etc/kubernetes/ssl")
```

Comment:
Kind of a note for myself. This is for testing. At the end the operator should read certificate TPOs from Kubernetes, right? So the cert-operator will create the TPOs and the aws-operator will consume them.